### ISSUE TITLE
Element sizes are not the set ones.

### ISSUE THREAD

**Issue by ebordon** [electron/electron#10047]
It happens to me using electron-forge. I've opened an [issue there ](https://github.com/electron-userland/electron-forge/issues/264#issuecomment-315423964) and they suggested to open an issue here. 
I also, tried by changing the scale factor and adding chrome flags but nothing changed.

* Electron version: electron-forge 3.0.3, electron-compile 6.4.1, electron-prebuilt-compile 1.6.11
* Operating system:  Windows 10, Ubuntu 14.04

### Expected behavior

Show a window with size 800x600. Render a red square with size 800x600.

All the window is red.

### Actual behavior

The red square size is less than expected.

### How to reproduce
 ```
  $ git clone https://github.com/ebordon/electron-scale-bug.git
  $ npm install
  $ electron-forge start
```

**Comment by jkleinsc** [comment]
@ebordon I am not able to reproduce the behavior you are describing using the repo you have provided.  I did notice that your example doesn't set the window to a set size (eg 800x600) but rather sets the window to the display height and width.  When I change your code to use a window size of 500x500, I see a the red box fill the whole window (regardless of my resolution).

A couple of suggestions:
1. Can you provide screenshots/screencasts of what you are seeing?
2. Can you try reproducing the issue in a chrome browser?

**Comment by ebordon** [comment]
I've tried to analyze a little more the issue, but I couldn't reproduce the described behavior. The real problem is that $(window) and $(document) width and height took different values if i run from electron-forge start or run from the compiled .exe generated by electron-forge package. 

After doing many tries, I figured out that if I move the out folder (generated by electron-forge package) outside the project dir, it works properly. I don't know where this weird problem is, but I have a workaround.
